README.md

```diff
@@ -110,8 +110,10 @@ Also accompanying modes and param types, as well as default values, are exported
   - [`sendMessageLead`](https://carforyou-service.preprod.carforyou.ch/swagger-ui/index.html#/Lead/createMessageLeadUsingPOST)
   - [`fetchDealerMessageLeads`](https://reporting-service.preprod.carforyou.ch/swagger-ui/index.html#/Reporting/getAllDealerMessageLeadsUsingGET)
   - [`fetchDealerCallLeads`](https://reporting-service.preprod.carforyou.ch/swagger-ui/index.html#/Reporting/getAllDealerCallLeadsUsingGET)
+  - [`fetchDealerWhatsAppLeads`](https://reporting-service.preprod.carforyou.ch/swagger-ui/index.html#/Reporting/getAllDealerWhatsAppLeadsUsingGET)
```

Comment:
there is already an entry in the README for `postWhatsappTrackingEntry` could you please move it up there?

Reply:
not sure if it is a good idea, we have the same structure for `postCallTrackingEntry` as well.

Reply:
yes but BE also groups them together and it's part of the reporting service. But up to you